python/singa/autograd.py

```diff
@@ -631,16 +712,39 @@ def backward(self, dy):
 
 
 def add_bias(x, b, axis=0):
+    """
+    Add Bias to each row / column of the Tensor, depending on the axis arg.
+    Args:
+        x: matrix.
+        b (CTensor): bias to be added.
```

Comment:
pls try to make the docstring consistent.
e.g., some variables have (CTensor) whereas some do not have.

Reply:
fixed, and I have checked the whole file again.